web/handlers.go

```diff
@@ -133,8 +134,13 @@ func generateDevCSP(c Context) string {
 
 	// Add flags for Webpack dev servers used by other products during development
 	if model.BuildNumber == "dev" {
-		// Focalboard runs on http://localhost:9006
-		devCSP = append(devCSP, "http://localhost:9006")
+		boardsURL, ok := os.LookupEnv("MM_BOARDS_DEV_SERVER_URL")
```

Comment:
We usually do `os.Getenv` and check for empty value. `os.LookupEnv` is only useful when you want to distinguish between unset values and empty values which is not important here. But not a big deal either ways.

Reply:
I'll change that for consistency. I think the last time I worked with environment variables was adding the ability for config values to be set to the empty string, so that's probably why I defaulted to `os.LookupEnv`